src/components/TweetAuthor.js

```diff
@@ -1,22 +1,27 @@
 import React, { PropTypes } from 'react';
-import './TweetAuthor.css';
+import './style/TweetAuthor.css';
 
 const propTypes = {
-  user: PropTypes.object.isRequired
+  user: PropTypes.object.isRequired,
+  showModal: PropTypes.func.isRequired
 }
 
 function TweetAuthor(props) {
+  function onClick(){
+    props.showModal(props.user)
+  }
   return (
     <div className="TweetAuthor">
```

Comment:
A slight usability improvement. It wasn't clear that the tweet authors were clickable. You can add a `cursor: pointer;` style for the TweetAuthor class to indicate it's clickable on hover. Same for the modal overlay.